creator-node/src/routes/audiusUsers.js

```diff
@@ -84,6 +84,8 @@ module.exports = function (app) {
         return errorResponseServerError(`Could not save to db: ${e}`)
       }
 
+      issueAndWaitForSecondarySyncRequests(req)
```

Comment:
doesn't this wait for at least one secondary sync confirm?

Reply:
i didnt await the function call

Reply:
ohhhh right 🤦

Reply:
this is kinda subtle b/c the function name is `wait`
can we add a comment for things like this in future? @SidSethi

Reply:
good call, adding now